addOns/webuipoc/src/main/pocs/reactWebUI/package.json

```diff
@@ -37,6 +45,12 @@
     ]
   },
   "devDependencies": {
+    "@eslint/js": "^9.2.0",
+    "eslint": "^8.57.0",
```

Comment:
Why two major versions?

Reply:
npm update might've added it, removed the older version

Reply:
There's also a config that should be removed.

Reply:
removing config too will effect lint functionality? as i believe unlike dependencies config wasn't duplicated, if I'm getting it wrong can u please mention that line to be removed

Reply:
One of the config files was for the older version.

Reply:
Removed

Reply:
Eslint Removed